<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>ProjectCreationDialog</class>
  <widget class="QDialog" name="ProjectCreationDialog">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>390</width>
        <height>275</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>Noggit Project Creation</string>
    </property>
    <property name="windowIcon">
      <iconset>
        <normalon>:/icon</normalon>
      </iconset>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
        <rect>
          <x>10</x>
          <y>40</y>
          <width>371</width>
          <height>181</height>
        </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
        <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
              <widget class="QLabel" name="label_2">
                <property name="text">
                  <string>Project Name</string>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QLineEdit" name="project_name"/>
            </item>
          </layout>
        </item>
        <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
              <widget class="QLabel" name="label_3">
                <property name="text">
                  <string>Game Client Directory</string>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QLineEdit" name="game_client_apth"/>
            </item>
            <item>
              <widget class="QPushButton" name="button_folder_select">
                <property name="text">
                  <string>...</string>
                </property>
              </widget>
            </item>
          </layout>
        </item>
        <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
              <widget class="QLabel" name="label">
                <property name="text">
                  <string>Project Expansion</string>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QComboBox" name="project_expansion">
                <item>
                  <property name="text">
                    <string>Wrath Of The Lich King</string>
                  </property>
                </item>
                <item>
                  <property name="text">
                    <string>Shadowlands</string>
                  </property>
                </item>
              </widget>
            </item>
          </layout>
        </item>
      </layout>
    </widget>
    <widget class="QPushButton" name="button_cancel">
      <property name="geometry">
        <rect>
          <x>310</x>
          <y>230</y>
          <width>71</width>
          <height>31</height>
        </rect>
      </property>
      <property name="text">
        <string>Cancel</string>
      </property>
    </widget>
    <widget class="QPushButton" name="button_ok">
      <property name="geometry">
        <rect>
          <x>240</x>
          <y>230</y>
          <width>61</width>
          <height>31</height>
        </rect>
      </property>
      <property name="text">
        <string>Ok</string>
      </property>
    </widget>
  </widget>
  <resources/>
  <connections/>
</ui>
